test/ruby/test_compile_prism.rb

```diff
@@ -167,7 +167,8 @@ def test_ConstantPathTargetNode
     end
 
     def test_ConstantPathWriteNode
-      # test_prism_eval("Prism::YCT = 1")
+      test_prism_eval("Prism::ConstantPathWriteNode= 1")
```

Comment:
I would make this PCT otherwise it's going to conflict when they run the tests more than once. 

And for below ::Prism::PCT